### ISSUE TITLE
Accessibility testing

### ISSUE THREAD

**Issue by nacnudus** [moj-analytical-services/shinyGovstyle#15]
This is brilliant, I wish I'd known about this months ago!  I've added a link to the [govdown](https://ukgovdatascience.github.io/govdown/) website.

Have you tested the accessibility using any of the resources from e.g. the GDS empathy lab?  If not, I'd be happy to help.  Here's are the [results for govdown](https://github.com/ukgovdatascience/govdown/issues/11).

**Comment by rosswyatt** [comment]
Interesting.  At the minute, I've assumed that as individual components, they are accessible as hard to evaluate when used in different setups, which is a bit of a cope out on my part...

So from what I can tell, you have used govdown to create a site then tested accessibility?

**Comment by nacnudus** [comment]
Yes, that's what I did with govdown.  Of course that doesn't guarantee usability of sites themselves, but it was still worth doing to reveal one glitch:

> There was a problem with invisible focus in code blocks, which has been fixed by disabling code highlighting (see #45).
